dir.props

```diff
@@ -151,7 +151,7 @@
 
     <ProjectUrl>https://dot.net</ProjectUrl>
 
-    <!-- PreReleaseSuffix for TFS win8-arm CoreCLR package -->
+    <!-- PreReleaseSuffix for packages published from closed build (e.g. CoreCLR for Arm32, APISet, etc) -->
     <ExternalExpectedPrerelease>rc4-24220-00</ExternalExpectedPrerelease>
```

Comment:
Leaving this number as-is will create invalid dependencies.  I primed myget with packages that have version beta-24220-00.  Can you use this instead?